pkg/utils/kubernetes/mock/gomock_reflect_349464534/prog.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
Remove this file and also `pkg/utils/kubernetes/mock/gomock_reflect_349464534/prog.bin`